client/extensions/woocommerce/app/order/order-created.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import Gridicon from 'gridicons';
+import { localize, moment } from 'i18n-calypso';
```

Comment:
`moment` is also [passed as a prop](https://github.com/Automattic/i18n-calypso#localize) from calling `localize`, like `translate`, so you don't need to import it here.